### DISCUSSION TITLE
Button icons/markings not showing up in Windows7

### DISCUSSION THREAD

**Discussion by skycamel** [rern/rAudio#1159] [General]
Since a few updates the icons/markings on the buttons (play/stop/+/-/next....) don't show up anymore in the WebUI (both in Firefox and Chrome), they just stay blank.
However, in Android and Linux there's no issue. Does anyone has an idea how I can fix that?
It's kind of annoying to have to guess where the buttons are.
Cheers,
Luc

**Comment by hjfisch** [comment]
I'm using Vivaldi , Waterfox , Firefox and sometimes Chrome and cannot confirm this bug . Wether in Windows nor in Linux.

**Comment by skycamel** [comment]
this is how it looks ...

[rAudio_WebUI.pdf](https://github.com/rern/rAudio-1/files/10298239/rAudio_WebUI.pdf)

**Comment by hjfisch** [comment]
![Screenshot 2022-12-24 102446](https://user-images.githubusercontent.com/93144100/209429587-548bda54-814f-426f-8104-b304fb34b1b7.jpg)
Seems not to be a problem with r-audio .

**Reply by skycamel** [reply]
Yes I'm sure it's not rAudio as it works on my Linux. But I thought (hoped) that someone had the same issue and found a solution :-)
And it used to be fine, it started with update 20220610 I think and all others after that. Not sure what changed.
When I revert back to the older release it's working fine again, so it's not something that changed in my windows setup.
